### ISSUE TITLE
Overfit test set?

### ISSUE THREAD

**Issue by ThibaultGROUEIX** [NVIDIAGameWorks/kaolin#169]
Hi,
In Image_Mesh_Recon_Direct example, the model that performs best in the test set is used for evaluation. Isn't this  biased? It is also the case in Geometric, Pixel2mesh...
In [eval.py](https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/ImageRecon/GEOmetrics/eval.py#L70), it might be best to replace "best" by "recent",
Cheers,
Thibault

**Comment by krrish94** [comment]
Sorry for the overloaded terminology. [eval.py](https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/ImageRecon/GEOmetrics/eval.py) currently only runs on the validation set. For the official test split, one will need to change [this line](https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/ImageRecon/GEOmetrics/eval.py)
